exercise/models.py

```diff
@@ -1,9 +1,8 @@
 from django.db import models
 
+from picklefield.fields import PickledObjectField
```

Comment:
I wouldn't recommend using a pickled objects because it makes migrations a real pain. Using a JSON field would be a better choice: https://django-mysql.readthedocs.io/en/latest/model_fields/json_field.html